client/frontend/src/views/UserView.vue

```diff
@@ -176,6 +180,9 @@ function togglePermission(scope) {
           :error="passwordError"
           @blur="passwordError = (validate.password(password) || password.length === 0) ? '' : t('error.PasswordInvalid')"
         />
+        <div v-if="$api.auth.hasScope('users.perms.view')">
+          <h2>{{ t('users.Authentication') }}: {{ otpactive ? t('users.TwoFactor') : t('users.Password') }}</h2>
```

Comment:
I think this makes more sense to just be "2FA Enabled: Yes/No" instead of it being "Authentication".
I had ideas in the past to add OAuth support to some extent, which would mean this title would be an issue.

Reply:
Adjusted in [d4ae15c](https://github.com/pufferpanel/pufferpanel/pull/1374/commits/d4ae15c07728b35c96a29ccea9a790071dfafdd4)

Also corrected my 'TwoFactor' naming convention to 'Otp' to match some of the other language variables.